src/tsickle.ts

```diff
@@ -385,7 +385,8 @@ class ClosureRewriter extends Rewriter {
     for (const heritage of decl.heritageClauses!) {
       if (!heritage.types) continue;
       if (decl.kind === ts.SyntaxKind.ClassDeclaration &&
-          heritage.token !== ts.SyntaxKind.ImplementsKeyword) {
+          heritage.token !== ts.SyntaxKind.ImplementsKeyword &&
+          !hasModifierFlag(decl, ts.ModifierFlags.Ambient)) {
```

Comment:
I was confused by this because I had forgotten that this loop is collecting up jsdoc.  I think the reason for this extra check here is that externs do not use ES6 classes, right?  Maybe put a comment at the top of the loop or here?